### DISCUSSION TITLE
Switching underlying voronoi library from `d3-voronoi` to `d3-delaunay`.

### DISCUSSION THREAD

**Discussion by Californian** [airbnb/visx#1272] [Ideas]
Might it make sense to switch the underlying voronoi library from `d3-voronoi` to `d3-delaunay`?

According to the `d3-voronoi` package README linked in this library, the package is deprecated with the following notice:
>Deprecation notice: Consider using the newer d3-delaunay instead of d3-voronoi. Based on Delaunator, d3-delaunay is 5-10× faster than d3-voronoi to construct the Delaunay triangulation or the Voronoi diagram, is more robust numerically, has Canvas rendering built-in, allows traversal of the Delaunay graph, and a variety of other improvements.

**Comment by williaster** [comment]
Ah, yes this could make sense. maybe we could add a new `@visx/delaunay` package, then deprecate `@visx/voronoi`?

Happy to review a PR if someone is interested in adding this 👍

**Reply by williaster** [reply]
tracking this with https://github.com/airbnb/visx/issues/1330
